docs/model-dev-guide/api-guides/apis-howto/deepspeed/deepspeed.rst

```diff
@@ -308,8 +308,62 @@ interleaving micro batches:
 Fully Sharded Data Parallelism (FSDP)
 =====================================
 
-To use FSDP, use the PyTorch FSDP package as usual along with the Core API. ``PytorchTrial`` API
-does not support FSDP. To find out more about the PyTorch FSDP package, visit `PyTorch tutorials
+To use FSDP, use the PyTorch FSDP package as usual along with the :ref:`api-core-ug`.
+
+.. note::
+
+   ``PytorchTrial`` API does not support FSDP.
+
+Example of a simple transformer model adapted from GPT-fast training on sample data:
```

Comment:
Since the example below is generic, I would recommend not referencing GPT-fast here (also many may not be familiar).

Reply:
Could preface the example by saying we have a generic `Transformer` model build from `TransformerBlock`, `Embedding` and `LMHead` layers which are each wrapper in their own FSDP unit. Probably detailed enough.